README.md

```diff
@@ -208,7 +211,7 @@ logging.getLogger(__name__).addHandler(clp_handler)
 
 ## Compatibility
 
-Tested on Python 3.6, 3.8, and 3.11 (should also work on newer versions).
+Tested on Python 3.8 and 3.11 (should also work on newer versions).
```

Comment:
I think we can edit/reword this section to explicitly say our minimum supported version is 3.7. We should probably test on 3.7 considering that is the minimum required version in pyproject.toml.

Reply:
We don't need to explicitly state the minimum supported version. It will be shown in the PyPI project home page :D